omnibus/config/projects/agent.rb

```diff
@@ -322,6 +325,15 @@
     end
   }
 
+  raise_if_fips_symbol_not_found = Proc.new { |symbols|
+      count = symbols.scan("github.com/microsoft/go-crypto-winnative").count()
+      if count == 0
+        raise FIPSSymbolsNotFound.new("Expected to find symbol 'github.com/microsoft/go-crypto-winnative' but no symbol was found.")
+      else
+        log.info(log_key) { "Symbol 'github.com/microsoft/go-crypto-winnative' found #{count} times in binary." }
```

Comment:
I believe `log_key` does not exist. What If see do in other files is `log.info(self.class.name)` I'm not sure what would `self.classname` would be in this context to be honest 😄

Reply:
We have this in the CI `[Builder: datadog-agent] I | 2024-09-27T09:08:26+00:00 | Symbol '_Cfunc_go_openssl' found 748 times in binary ''.`. I'm not sure but maybe `log_key` is defined taken from [here](https://github.com/DataDog/omnibus-ruby/blob/db96ee7e2a9269ab864f618d7c447bc22e107fff/lib/omnibus/software.rb#L1441-L1443) ?

Reply:
(I need to fix the message to include the binary tho)

Reply:
Yep, it comes from omibus-ruby. In this case most likely the one defined in `project.rb` since we're in a project scope

Reply:
Sure, definitely not blocking :)